code/game/machinery/status_display.dm

```diff
@@ -175,6 +224,15 @@
 	dir = EAST
 	pixel_x = -32
 
+//makes it go on the wall when built
+/obj/machinery/status_display/Initialize(mapload, ndir, building)
```

Comment:
Are you sure ndir and building are passed in? It doesn't look like it.

Reply:
![image](https://user-images.githubusercontent.com/85910816/135733333-651e0b49-9015-4010-b2bf-2e9674735077.png)
without this, it places on the tile above where you're standing